plaidml/op/op_test.cc

```diff
@@ -251,6 +251,22 @@ TEST_F(OpTest, Elu) {
   runProgram(program);
 }
 
+TEST_F(OpTest, ExplicitPadding) {
+  auto I = Placeholder(DType::FLOAT32, {2, 3}, "A");
+  auto O = op::explicit_padding(I, {2, 1}, {2, 1}, op::PadMode::CONSTANT, -1.0);
+  auto program = makeProgram("explicit_padding", {O});
+
+  std::vector<float> I_input = {1, 2, 3,  //
+                                4, 5, 6};
+  std::vector<float> O_output = {-1, -1, -1, -1, -1,  //
+                                 -1, -1, -1, -1, -1,  //
+                                 -1, 1,  2,  3,  -1,  //
+                                 -1, 4,  5,  6,  -1,  //
+                                 -1, -1, -1, -1, -1,  //
+                                 -1, -1, -1, -1, -1};
```

Comment:
Minor nitpick: did you run `clang-format` here? Looks like the 3rd and 4th rows are offset from the rest of the matrix.

Reply:
This is actually what `clang-format` changed it to, I guess it tries to keep the fronts of the numbers aligned instead of the commas

Reply:
Interesting, thanks for clarifying. I see it in other places too.